packages/shared/src/components/comments/AdAsComment.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { ReactElement, useContext, useEffect } from 'react';
+import { useQuery } from '@tanstack/react-query';
+import { Ad } from '../../graphql/posts';
+import { apiUrl } from '../../lib/config';
+import { TruncateText } from '../utilities';
+import AdLink from '../cards/AdLink';
+import { adLogEvent } from '../../lib/feed';
+import LogContext from '../../contexts/LogContext';
+import { ProfileImageSize, ProfilePicture } from '../ProfilePicture';
+import PlaceholderCommentList from './PlaceholderCommentList';
+
+export const AdAsComment = (): ReactElement => {
+  const { logEvent } = useContext(LogContext);
+  const ad = useQuery<Ad>(
+    ['ads', 'post'],
+    async () => {
+      const res = await fetch(`${apiUrl}/v1/a/post`);
+      const ads: Ad[] = await res.json();
+      return ads[0];
+    },
+    {
+      enabled: true,
```

Comment:
Maybe we can add some time here?

Reply:
Changed the query keys a bit, added postId so now we get a new ad on every post. Checked with Ido about any stale time but he didn't have anything specific in mind, so added 1h, we can change later if we want